### PR TITLE
Update eslint config, remove useless configuration

### PR DESCRIPTION
**Pull request by AlexeyKorkoza**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "dev:watch": "babel-watch ./src/app",
-    "eslint:backend": "eslint src",
+    "eslint:backend": "eslint ./src",
```
Comment (dkryvi) [inline package.json]: `eslint src` doesn't work correctly?

Comment (AlexeyKorkoza) [reply package.json]: This command worked correctly. I added './' by mistake.

Comment (dkryvi) [reply package.json]: Return back

Comment (AlexeyKorkoza) [reply package.json]: One moment

Comment (AlexeyKorkoza) [reply package.json]: I returned back

**Pull request merged**